From 8952ce7f2fb9b5a5deefcbf68c2e9cd1c0b87be7 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:44 -0400
Subject: [redhat] kabi: Adding symbol blk_queue_segment_boundary (block/blk-settings.c)

Message-id: <4fc5395ae089c7857d96ab569e4e2184568aa4b6.1410510773.git.poros@redhat.com>
Patchwork-id: 96111
O-Subject: [PATCH RHEL7.1 044/233] kabi: Adding symbol blk_queue_segment_boundary (block/blk-settings.c)
Bugzilla: 1062361
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to copy the characteristics of the bottom queue to top (PowerPath) queue.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/blk_queue_segment_boundary
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/blk_queue_segment_boundary
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/blk_queue_segment_boundary

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_segment_boundary  b/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_segment_boundary 
new file mode 100644
index 0000000..c81cd2a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/blk_queue_segment_boundary  
@@ -0,0 +1,2 @@
+#1-
+0xf4dcc005 blk_queue_segment_boundary vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/blk_queue_segment_boundary  b/redhat/kabi/kabi-module/kabi_s390x/blk_queue_segment_boundary 
new file mode 100644
index 0000000..e9dab2c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/blk_queue_segment_boundary  
@@ -0,0 +1,2 @@
+#1-
+0xfca65b1f blk_queue_segment_boundary vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_segment_boundary  b/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_segment_boundary 
new file mode 100644
index 0000000..014507e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/blk_queue_segment_boundary  
@@ -0,0 +1,2 @@
+#1-
+0xeedc4654 blk_queue_segment_boundary vmlinux EXPORT_SYMBOL
-- 
1.7.1